dumpling/export/config.go

```diff
@@ -555,6 +558,9 @@ const (
 	defaultDumpGCSafePointTTL = 5 * 60
 	defaultEtcdDialTimeOut    = 3 * time.Second
 
+	DefaultCollationCompatible = "loose"
```

Comment:
Default against Strick is so weird.
```suggestion
	LooseCollationCompatible = "loose"
```

Reply:
ok